packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -265,6 +265,30 @@ class TwinDeploymentHandler {
     return deployments;
   }
 
+  async checkFarmIps(twinDeployments: TwinDeployment[]) {
+    const farms: { [key: string]: { ips: number } } = {};
```

Comment:
why that complex type? it can be Map<number, number>

And please use a better name for the variable